src/libsat355.h

```diff
@@ -0,0 +1,56 @@
+#ifndef LIBSAT355_H
+#define LIBSAT355_H
+
+#include <stdio.h>
+#include "dllmain.h"
+
+// TRICKY: Make sure only C++ compiles this.
+// When compiled from C or SwiftUI, it is excluded
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum ErrorCode
+{
+    kOK = 0,
+    kInvalidTLE,
+    kInvalidTime,
+    kInternalError
+};
```

Comment:
after reviewing the c-abi spec, i think we'll need to adjust this because:
"The C standard allows compilers to choose the underlying integer type as they see fit. For example GCC always uses `int` as long as all enum values fit in 32bits. However, another compiler vendor could use `char` when encountering the exact same enum provided all the values fit within 8bits"

long story, shot:
- you should use `int` as the return value, instead of `ErrorCode`